Demos/02-angular-connect-rest/README.md

```diff
@@ -19,7 +19,7 @@ There are many sample applications that demonstrate how to use the Microsoft Gra
 You will use a QuickStart application to demonstrate working with AngularJS and the Azure AD v2 endpoint.
 
 1. Download/clone the [Microsoft Graph connect sample for AngularJS](https://github.com/microsoftgraph/angular-connect-rest-sample) and open it in a code editor of your choice.
-    >Note: This solution requires that you've installed Node.js. Please see the prerequisites in the README.md file for more information.
+    >Note: This solution requires that you've installed Node.js. Please see the prerequisites in the sample project's [README.md](https://github.com/microsoftgraph/angular-connect-rest-sample/blob/master/README.md) file for more information.
```

Comment:
This sample is going away soon. Can you point folks to https://docs.microsoft.com/graph/tutorials/angular instead?